app.js

```diff
@@ -45,18 +46,34 @@ function displayPhrases() {
 
     for (const phrase of character.phrases) {
         // > create an <li> element
+        const li = document.createElement('li');
         // > set the text of the li to the phrase
+        li.textContent = phrase;
         // > append the <li> to the phrases list
+        phraseList.append(li);
     }
 }
+function isEmpty() {
+    if (phraseInput.value === '') {
+        alert('Phrase cannot be empty');
```

Comment:
Awesome job with the alert! You could move this into an else statement on line 76.5 if you want!